state/store.go

```diff
@@ -54,6 +55,14 @@ func abciResponsesKey(height int64) []byte {
 	return encodeKey(prefixABCIResponses, height)
 }
 
+func stateKey() []byte {
+	res, err := orderedcode.Append(nil, prefixState)
+	if err != nil {
+		panic(err)
+	}
+	return res
+}
```

Comment:
FYI: I missed this when I first implemented key encoding so just adding it now. It's something we set and forget when the state store is created so I added as part of the actual struct. Not sure if this is exactly the most idiomatic way to do it